<launch>
  <node name="robot_command_server" pkg="robot_control" type="Server" output="screen">
    <param name="roboter_ip" value="192.168.12.10" type="str" />
    <param name="roboter_port" value="29999" type="int" />
    <param name="program_name" value="experiment_20191205.urp" type="str" />
  </node>
</launch>

